docs/installation.md

```diff
@@ -5,15 +5,15 @@
 Once you have Node.js and NPM installed on your machine you're ready to download and install the SMK-CLI. Simply type the following into your command line of choice:
 
 ```bash
-npm install —global smk-cli
+npm install —global @bcgov/smk-cli
 ```
 
-If you prefer to install a specific version, you can check NPM for the available version list in the [NPM repository](https://www.npmjs.com/package/smk-cli/)
+If you prefer to install a specific version, you can check NPM for the available version list in the [NPM repository](https://www.npmjs.com/package/@bcgov/smk-cli)
 
 then append the version tag on your install command:
 
 ```bash
-npm install —global smk-cli@1.0.0-beta.6
+npm install —global @bcgov/smk-cli@1.0.0-beta.6
```

Comment:
Could you please remove this reference to the beta version, the line can just be:
npm install --global @bcgov/smk-cli

Reply:
Sure!